text/0000-fix-fees.md

```diff
@@ -0,0 +1,180 @@
+- Title: fix-fees
+- Authors: [John Tromp](mailto:john.tromp@gmail.com)
+- Start date: August 23, 2020
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000)
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+## Summary
+[summary]: #summary
+
+Change Grin's minimum relay fees to be weight proportional and make output creation cost about a Grin-cent.
+Restrict fees to 40 bits, using 4 of the freed up 24 bits to specify the desired tx priority,
+and leaving the remaining 20 bits for future use.
+NOTE: this is a hard-forking change.
+
+## Motivation
+[motivation]: #motivation
+
+The former fee requirements suffer from being somewhat arbitrary, and not miner incentive compatible.
+They are not even linear; to avoid a negative minimum fee, they are rounded up to zero.
```

Comment:
I believe we use a value of `1 * BASE_FEE` here for minimum fee, rather than 0.

Reply:
fixed